OutOfSchool/OutOfSchool.WebApi/Models/Workshop/WorkshopDTO.cs

```diff
@@ -116,7 +116,7 @@ public class WorkshopDTO : IValidatableObject
     [ModelBinder(BinderType = typeof(JsonModelBinder))]
     public AddressDto Address { get; set; }
 
-    public List<TeacherDTO> Teachers { get; set; }
+    public List<TeacherDTO> Teachers { get; set; } = new ();
```

Comment:
I think you should check this property for null value in place where it uses and throw an exeption if it's null

Reply:
If I remember right, actually we allow to create workshops without teachers, so an exception if null may be inappropriate behaviour